.gitignore

```diff
@@ -4,3 +4,4 @@ dwl
 *-protocol.h
 .ccls-cache
 config.h
+build
```

Comment:
https://mesonbuild.com/FAQ.html#how-do-i-ignore-the-build-directory-in-my-vcs

This is not necessary with current versions of Meson.

You can probably delete the .gitignore altogether if it removes the Makefile which is the only thing creating those files...